### DISCUSSION TITLE
Plot tree

### DISCUSSION THREAD

**Discussion by esfingers** [pycaret/pycaret#3421] [Q&A]
When I start to plot tree, it turns out like this. I want to know how to fix this.

`~\anaconda3\lib\site-packages\pycaret\internal\pycaret_experiment\tabular_experiment.py in tree()
   1509                             i: class_name
   1510                             for i, class_name in enumerate(
-> 1511                                 get_label_encoder(self.pipeline).classes_
   1512                             )
   1513                         }

AttributeError: 'NoneType' object has no attribute 'classes_'`

**Comment by Jogsterdgl** [comment]
I am having the same problem as you. I submitted an issue for it. Form my perspective it is a bug, i am not 100% sure tough.  
Here is the issue if you are interested: https://github.com/pycaret/pycaret/issues/3444 .
